static/dashboard/css/dashboard.css

```
@@ -0,0 +1,210 @@
+  --surface: #161b22;
+  --surface-2: #1c2128;
+  --border: #30363d;
+  --green: #3fb950;
+  --yellow: #d29922;
+  --red: #f85149;
+  --blue: #58a6ff;
+  --purple: #bc8cff;
+  --gray: #8b949e;
+  --text: #e6edf3;
+  --text-dim: #8b949e;
+  --font: -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.9.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)